eng/scripts/FinishDumpCollectionForHangingBuilds.ps1

```diff
@@ -0,0 +1,74 @@
+param(
+  [Parameter(Mandatory = $true)]
+  [ValidateNotNullOrEmpty()]
+  [string]
+  $ProcDumpDumpFolderPath
+)
+
+Write-Output "Finishing dump collection for hanging builds.";
+
+$repoRoot = Resolve-Path "$PSScriptRoot\..\..";
+$ProcDumpDumpFolderPath = Join-Path $repoRoot $ProcDumpDumpFolderPath;
+
+$sentinelFile = Join-Path $ProcDumpDumpFolderPath "dump-sentinel.txt";
+if ((-not (Test-Path $sentinelFile))) {
+  Write-Output "No sentinel file available in '$sentinelFile'.";
```

Comment:
Suggest something like "StartDumpCollectionForHangingBuilds.ps1 has not been executed".